src/networkWrapper/protocols/socketio.js

```diff
@@ -49,36 +43,85 @@ class SocketIO extends RTWrapper {
   }
 
   /**
-   * Registers a callback on a room. Once 1 message is received, fires the
+   * Registers a callback on an event. Once 1 message is received, fires the
    * callback and unregister it afterward.
    *
-   * @param {string} roomId
+   * @param {string} event
    * @param {function} callback
    */
-  once(roomId, callback) {
-    this.socket.once(roomId, callback);
+  addOnceListener(event, callback) {
```

Comment:
is it necessary to redefine these methods ?

Reply:
No, it is not. I removed it along with a few others (`on`, `prependOnceListener`, ...)